### PR TITLE
Release v0.3.0rc2

### PR DESCRIPTION
**Pull request by stes**
- [x] Make sure all tests pass on the rc branch. Also make sure that the [milestone](https://github.com/stes/neural_cl/milestones) related to this release is fully done. Move issues that wont make it into the release to the next milestone, then close the milestone.
- [x] Head to [`cebra.__init__`](neural_cl/cebra_public/cebra/__init__.py) and make sure that the `__version__` is set correctly.
- [x] [Create a PR](https://github.com/stes/neural_cl/compare) between rc and `main`
- [ ] Tag the PR with the `release` [label](https://github.com/stes/neural_cl/issues/labels).
- [ ] A [github action will be run ](https://github.com/stes/neural_cl/blob/main/.github/workflows/internal-release.yml) -- if it doesnt start, try removing and re-adding the release label (step 4).
- [ ] Carefully check that the release looks fine --- the version from the PR will be pushed to [`release-staging`](https://github.com/AdaptiveMotorControlLab/cebra-internal/tree/release-staging) and [`staging`](https://github.com/AdaptiveMotorControlLab/cebra-internal/tree/staging) in the [`cebra-internal`](https://github.com/AdaptiveMotorControlLab/cebra-internal) repo. *Note: If you update the PR, these version will not be automatically updated. Repeat step 4 or trigger a manual workflow run if you need to update the staging version*
- [ ] If all looks good, tests pass and the PR is reviewed, merge the PR **using rebase merging**.
- [ ] Delete the branch
- [ ] Checkout the updated `main` branch, `git tag v1.2.3` with the correct format (if alpha/beta tags are used in __version__, use `v1.2.3a4` or `v1.2.3b4`), and `git push v1.2.3` the tag.
- [ ] Pushing the tag will update the release in `cebra-internal`. The source tree will land on [`main`](https://github.com/AdaptiveMotorControlLab/cebra-internal), the pre-build wheel and source distribution on [`release`](https://github.com/AdaptiveMotorControlLab/cebra-internal/tree/release).

### CODE REVIEW

**Comment by stes** [comment]
Released on test pypi: https://test.pypi.org/project/cebra/0.3.0/

**Comment by stes** [comment]
GPU tests on datasets pass:

image

**Review by MMathisLab — APPROVED** [review]

**Comment by stes** [comment]
Let's potentially wait for one more dataset-related PR from @gonlairo before merging.

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ build: dist
 archlinux:
 	mkdir -p dist/arch
 	cp PKGBUILD dist/arch
-	cp dist/cebra-0.3.0rc1.tar.gz dist/arch
+	cp dist/cebra-0.3.0.tar.gz dist/arch
```
Comment (stes) [inline Makefile:13]:
```suggestion
	cp dist/cebra-0.3.0rc2.tar.gz dist/arch
```

File: PKGBUILD [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: Steffen Schneider <stes@hey.com>
 pkgname=python-cebra
 _pkgname=cebra
-pkgver=0.3.0rc1
+pkgver=0.3.0
```
Comment (stes) [inline PKGBUILD:4]:
```suggestion
pkgver=0.3.0rc2
```

File: cebra/__init__.py [hunk 3]
```diff
@@ -47,7 +47,7 @@
 
 import cebra.integrations.sklearn as sklearn
 
-__version__ = "0.3.0rc1"
+__version__ = "0.3.0"
```
Comment (stes) [inline cebra/__init__.py:50]:
```suggestion
__version__ = "0.3.0rc2"
```

File: reinstall.sh [hunk 4]
```diff
@@ -15,7 +15,7 @@ pip uninstall -y cebra
 # Get version info after uninstalling --- this will automatically get the
 # most recent version based on the source code in the current directory.
 # $(tools/get_cebra_version.sh)
-VERSION=0.3.0rc1
+VERSION=0.3.0
```
Comment (stes) [inline reinstall.sh:18]:
```suggestion
VERSION=0.3.0rc2
```

**Comment by stes** [comment]
Waiting for https://github.com/AdaptiveMotorControlLab/CEBRA/pull/69

**Comment by stes** [comment]
Added a docker container for testing --- running full test suite now.

**Comment by stes** [comment]
All tests pass

image

cc @MMathisLab for review

**Pull request merged**